### ISSUE TITLE
Typescript/React errors showing up in the client when using VSCode

### ISSUE THREAD

**Issue by underbluewaters** [seasketch/geoprocessing#373]
I can't make any sense of this. It isn't causing any issues running storybook, but it does make it hard to tell if I have other type errors in my code since it flags many components up and down the file.

![Screenshot 2024-11-19 at 2 04 06 PM](https://github.com/user-attachments/assets/0664b8f7-dcd3-42e0-86b6-9fba1589c23d)

**Comment by twelch** [comment]
Can you give a little more insight. Is this code generated by create:report?  I don't think I've ever seen this error

**Comment by twelch** [comment]
how are you importing react in that file?

**Comment by twelch** [comment]
Consider blowing away node_modules and reinstalling, see if that helps.  Restart the vscode window.

**Comment by underbluewaters** [comment]
This is after running create:report and not modifying any code, just looking at the generated tsx files. I tried:
  1. restarting the typescript server
  2. deleting and reinstalling node_modules
  3. closing the VSCode window and restarting VSCode.

No luck so far. VSCode updated itself to 1.95.2 once I restarted it, but that didn't impact anything. Host system typescript is 5.3.3. Not sure that matters.

**Comment by underbluewaters** [comment]
Okay, I fixed it by re-adding this to the top:
```
import React from "react";
```

I forgot that I had removed it initially. Previously, I was getting an error related to that line so I removed it, but I don't have that error written down and it's not occurring now. I'm just going to close this ticket unless I have trouble when generating another component. I think there are still issues with a fresh install run-through but these details are probably misleading.
